membership-attribute-service/app/components/TouchpointComponents.scala

```diff
@@ -102,10 +96,9 @@ class TouchpointComponents(
 
   lazy val zuoraSoapService = {
     lazy val zuoraSoapClient =
-      new ClientWithFeatureSupplier(
+      new Client(
```

Comment:
we used to have this ClientWithFeatureSupplier class as a subclass of Client, but all the code was deleted from it recently as it was not needed, so I've flattened it out of the hierarchy